src/Adapter/Product/Combination/CommandHandler/UpdateCombinationDetailsHandler.php

```diff
@@ -107,6 +107,7 @@ private function fillUpdatableProperties(Combination $combination, UpdateCombina
 
         if (null !== $command->getWeight()) {
             $combination->weight = (float) (string) $command->getWeight();
+            $updatableProperties[] = 'weight';
```

Comment:
Is this change related?

Reply:
Yes and no, since the partial update didn't work we couldn't detect that this behat test was failing because everything was saved anyway So now that partial update works correctly the CI is falling here and prevents the merge
I could create yet another PR to fix this separately but it feels a bit overkill IMHO